modules/ROOT/pages/advanced-templates.adoc

```diff
@@ -79,6 +79,66 @@ To manage the template list
 ** Move all category items to another category.
 ** Delete template/category.
 
+[[read-only-templates]]
+=== Readonly templates
+
+include::partial$misc/admon-requires-7.4v.adoc[]
+
+This option sets a {pluginname} category to read-only in {productname}, streamlining template management. When set to read-only, users cannot perform actions such as moving, renaming, or adding new items to the category or its templates. A lock image
+/lock.svg[lock.svg] icon visually indicates that the templates are non-editable, and client-side restrictions prevent modification attempts from being submitted to the server.
+
+==== Data Structure
```

Comment:
@FarzadHayat , I don't know much about advance template plugin. I still think you are right because we don't have control of the integrator's backend. The lock feature looks like it only affect the UI.

Reply:
>The locked categories seems more like a user interface restriction.

While the integrator can add this, users could use `document.getElementById` to remove elements, or even an editor from the DOM, I doubt unless they have admin it will not update the DB IMO.

@tiny-ben-tran thoughts?